index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>McLaren</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  
+</head>
+
+  <body>
+  
+    <header>
+
+      <div class="headerLog">
+        <div class="headerLogFacebook"><i class="fa-brands fa-facebook-f"></i></div>
+        <div class="headerLogTwitter"><i class="fa-brands fa-twitter"></i></div>
+        <div class="headerLogLanguage"><a href="">English</a></div>
+        <div class="headerLogLanguage"><a href="">My Page</a></div>
+        <div class="headerLogout"><a href="">Logout</a></div>
+      </div>
+
+      <nav>
+        <div class="navLogo"><img src="images/mclarenlogo..png"></div>
```

Comment:
- [ ] Consider wrapping your logo in an `<a>` tag that links to the home page. This will make it easier to migrate to the home page from the about page.
- [ ] Kindly consider adding a Home button in the mobile menu to enable users to be able to migrate to the home page easily.
- [ ] Please add an `alt` attribute to all your images so that they can be displayed if the image is not loaded and also to aid screen readers know what the image is about. This is best practice for accessibility.